src/CSSList/CSSList.php

```diff
@@ -296,6 +296,22 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem The item.
+     * @param RuleSet|CSSList|Import|Charset $oSibling The sibling.
```

Comment:
IMHO the two "prose parts" of the PHPDoc comment are redundant with the parameter names, and I propose removing `The item.` and `The sibling.`.

Reply:
They are removed.